feature/about/src/test/java/io/github/droidkaigi/confsched2023/AboutScreenTest.kt

```diff
@@ -38,4 +38,20 @@ class AboutScreenTest {
             checkScreenCapture()
         }
     }
+
+    @Test
+    @Category(ScreenshotTests::class)
+    fun checkScrollShot() {
+        aboutScreenRobot {
+            setupAboutScreenContent()
+            scrollAboutScreen()
+            checkStaffAboutContentColumnCapture()
```

Comment:
Sorry for the confusion🙇 I mean like this

            setupAboutScreenContent()

            scrollAboutScreen()
            checkScreenCapture()

            scrollAboutScreen()
            checkScreenCapture()